tests/fixtures/meetup_response_example.html

```diff
@@ -0,0 +1,81 @@
+            <a class="eventCard--link" href="/PHP-The-Right-Way/events/123456/">
+                <span class="visibility--a11yHide">El título del meetup</span>
+            </a>
+            <div class="flex flex--column flex--spaceBetween">
+                <div class="flex-item flex-item--shrink">
+                    <div class="eventCardHead">
+                        <div class="flex flex--row">
+                            <div class="flex-item">
+                                <div class="eventTimeDisplay text--labelSecondary text--small wrap--singleLine--truncate margin--halfBottom color-viridian">
+                                    <time class="" dateTime="1595444400000">
+                                        <span class="eventTimeDisplay-startDate">
+                                            <span></span>,
+                                            <span class="eventTimeDisplay-startDate">
+                                                <span></span>
+                                            </span>
+                                        </span>
+                                    </time></div>
+                                <div class="text--ellipsisTwoLines text--sectionTitle margin--halfBottom">
+                                    <a class="eventCardHead--title" href="/PHP-The-Right-Way/events/123456/">El título del meetup</a>
+                                </div>
+                                <div class="flex flex--row flex--noGutters flex--alignCenter margin--halfBottom">
+                                    <div class="flex-item flex-item--shrink text--secondary">
+                                        <svg data-swarm-icon="true" height="24" width="24" viewBox="0 0 24 24" class="valign--middle"></svg>
+                                    </div>
+                                    <div class="flex-item valign--middle">
+                                        <div class="venueDisplay venueDisplay-venue padding--left-half text--secondary text--small">
+                                            <address>
+                                                <p class="wrap--singleLine--truncate">Online event</p>
+                                            </address>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="flex-item flex-item--shrink">
+                    <div>
+                        <div>
+                            <div>
+                                <p class="text--small padding--top margin--halfBottom">&nbsp;<br />&nbsp;<br />&nbsp;</p>
+                                <p class="text--small padding--top margin--halfBottom">La decripción del meetup</p>
```

Comment:
`decripción` > `descripción`

Reply:
> I added a minor comment.
> How will the event crawler be executed? Is there a command to be ran and that maybe shall be set in a cronjob?

The execution plan is not planned yet jeje, it could be performed as a cronjob each day in order to get the latest count of attendees.

Reply:
i remember asking @dmouse if it was possible to create cronjob on the server i remember he said yes, could you confirm @dmouse